src/main/java/com/shipmentEvents/handlers/EventHandler.java

```diff
@@ -63,6 +64,20 @@ public String handleRequest(ScheduledEvent scheduledEvent, Context context) {
         }
     }
 
+    // create a function that counts to 10
+    public void countToTen() {
+        for (int i = 0; i < 10; i++) {
+            System.out.println(i);
+        }
+    }
+
+    // create a function that has weak Security
+    public String weakEncryption(String message) throws Exception {
+        Cipher cipher = Cipher.getInstance("RSA");
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

It looks like your code uses a cipher object with an insecure transformation. To make your code more secure, use one of the following algorithms with a built-in integrity check: `AES/GCM/NoPadding` or `ChaCha20-Poly1305`.

[Learn more](https://cwe.mitre.org/data/definitions/327.html)